src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -404,6 +404,8 @@ public function load(array $configs, ContainerBuilder $container)
             ->addTag('controller.argument_value_resolver');
         $container->registerForAutoconfiguration(AbstractController::class)
             ->addTag('controller.service_arguments');
+        $container->registerForAutoconfiguration('Symfony\Bundle\FrameworkBundle\Controller\Controller')
```

Comment:
that one does not exist in 5.0, so it does not need to be restored.

Reply:
true, updated.